hummingbot/client/hummingbot_application.py

```diff
@@ -881,6 +885,10 @@ def start(self, log_level: Optional[str] = None):
                 strategy_logging_options = PureMarketMakingStrategyV2.OPTION_LOG_ALL
 
                 self.strategy = PureMarketMakingStrategyV2(market_infos=[self.market_info],
+                                                           number_of_orders= number_of_orders,
```

Comment:
Please don't put plugin arguments in the pure market making strategy's constructor. Once we've changed the architecture to plugins, then the way to do it is to give the arguments to plugin constructor and then give the plugins to the strategy.

Reply:
Note the compatibility arguments for this thing have been renamed "legacy_xxx" s.t. it can serve as a drop-in replacement for v1 in case the user doesn't want to use it in the plugin way. But that's the only place where this is allowed and it's well labeled.